services/dataset_service.py

```diff
@@ -21,3 +23,19 @@
 
 def get_datasets() -> List[dataset.Dataset]:
     return DATASETS
+
+def get_dataset(dataset_id) -> Union[dataset.Dataset, None]:
```

Comment:
Use a typehint for dataset_id.

Reply:
Use Optional[dataset.Dataset] instead of Union[dataset.Dataset, None].

Reply:
Done.